openjdk-tomcat-ubuntu-vm/azuredeploy.parameters.json

```diff
@@ -3,16 +3,10 @@
   "contentVersion": "1.0.0.0",
   "parameters": {
     "javaPackageName": {
-      "value": "openjdk-7-jdk"
+      "value": "openjdk-13-jdk"
```

Comment:
remove this and use a defaultValue in the template

Reply:
yes added default "

defaultValue": "openjdk-13-jdk",
            "allowedValues": [
              "openjdk-11-jdk",
              "openjdk-12-jdk",
              "openjdk-13-jdk"